tests/suites/0_stateless/README.md

```diff
@@ -22,6 +22,7 @@ xx_yyyy_[test_name]
 * 10 -- for Describe tests, [example](10_0000_describe_table.sql)
 * 11 -- for Show tables, [example](11_0000_show_tables.sql)
 * 12 -- for insert into select, [example](12_0000_insert_into_select.sql)
+* 13 -- for Show functions, [example](14_0000_show_functions.sql)
```

Comment:
We can merge all show to 11 `11 -- for Show commands`

Reply:
Updated